.gitignore

```diff
@@ -1,10 +1,34 @@
-# ignore all build folders
-**/build/
-**/node_modules/
-**/storybook-static/
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# build
+build
```

Comment:
the ** will make sure that it matches any subfolder build, etc.  I think the way you have it now will just ignore the top level folders.

Reply:
Tested. With a flat name it will get both.

Reply:
![Screen Shot 2021-10-05 at 8 25 36 AM](https://user-images.githubusercontent.com/10125297/136021909-36d46841-1f55-4f92-ad4e-27c6f1c8b3af.png)